optics-core/src/Optics/Getter.hs

```diff
@@ -47,6 +47,9 @@ import Optics.Internal.Profunctor
 type Getter s a = Optic' A_Getter NoIx s a
 
 -- | View the value pointed to by a getter.
+--
+-- If you want to 'view' a type-changing optic, you may need to use
```

Comment:
On the second thought: *a type-changing optic* doesn't sound right.

I'm not sure whether *an insufficiently polymorphic type-changing optic* would be any better, but I think less wrong?

Reply:
Yes, I wondered what to call these. Your suggestion is a bit better, and I've reworded it slightly further.